app/src/main/java/com/trueedu/project/ui/view/SettingFragment.kt

```diff
@@ -0,0 +1,108 @@
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.ComposeView
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.fragment.app.FragmentManager
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import com.treuedu.project.R
+import com.trueedu.project.Greeting
+import com.trueedu.project.ui.common.BasicText
+import com.trueedu.project.ui.common.DividerHorizontal
+import com.trueedu.project.ui.theme.TrueProjectTheme
+import com.trueedu.project.ui.topbar.TopBar
+
+class SettingFragment: BottomSheetDialogFragment() {
+    companion object {
+        fun show(
+            fragmentManager: FragmentManager
+        ): SettingFragment {
+            val fragment = SettingFragment()
+            fragment.show(fragmentManager, "setting")
+            return fragment
+        }
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setStyle(STYLE_NORMAL, R.style.FillScreenSheetTheme)
+    }
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        return super.onCreateDialog(savedInstanceState).apply {
+            (this as? BottomSheetDialog)?.behavior?.run {
+                this.skipCollapsed = true
+                this.state = BottomSheetBehavior.STATE_EXPANDED
+                this.isDraggable = false
+            }
+        }
+    }
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+        return ComposeView(requireContext()).apply {
+            layoutParams = ViewGroup.LayoutParams(
+                ViewGroup.LayoutParams.MATCH_PARENT,
+                ViewGroup.LayoutParams.MATCH_PARENT
+            )
+            setContent {
+                TrueProjectTheme {
+                    Scaffold(
+                        topBar = {},
+                        modifier = Modifier
+                            .fillMaxSize()
+                            .background(color = MaterialTheme.colorScheme.background),
+                    ) { innerPadding ->
+                        Column(
+                            modifier = Modifier.fillMaxSize().padding(innerPadding)
+                        ) {
+                            SettingItem("appkey 설정", {})
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+fun SettingItem(
+    text: String = "나의 설정",
+    onClick: () -> Unit = {},
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        modifier = Modifier
+            .fillMaxWidth()
+            .clickable { onClick() }
```

Comment:
```suggestion
            .clickable onClick
```

혹은

```suggestion
            .clickable(onClick)
```

로 표현할 순 없나요?

Reply:
안됩니다. onClick 하나만 파라미터 받는 게 아니라
마지막 파라미터만 callback method 로 받는 형식이라서요

Reply:
마지막 파라미터면서 function 인 경우만 쓸 수 있는 문법설탕이었군요. :ok_hand: